### ISSUE TITLE
Feature request: Mountain Passes

### ISSUE THREAD

**Issue by Maxszik** [nvkelso/natural-earth-vector#465]
I wish passes were a part of natural Earth. The Khyber Pass, The Brenner Pass, Torugart and so many others carry historical, cultural and contemporary importance. If we have peaks and capes in the dataset, for example, I think mountain passes deserve a place as well.

**Comment by Andrettin** [comment]
I definitely agree. And I think the project would also benefit from having forest geopolygons for the physical label geographical areas.

**Comment by nvkelso** [comment]
Great idea! Do you have any data or research time to contribute for mountain passes?

**Comment by Andrettin** [comment]
@nvkelso Would geocoordinates set out in 19th century books be a permissible source of information for the project (considering they are in the public domain)? I have some experience with researching them, so I could see if I manage to find something useful. I imagine this could be helpful to get the geocoordinates of points such as mountain passes and settlements.

**Comment by nvkelso** [comment]
Yes, that should be fair game. But we'd want to adjust to match the SRTM based relief, admin boundaries, and roads that are in Natural Earth.

Is there a source similar to https://peakbagger.com/ but for passes?

**Comment by ImreSamu** [comment]
Wikidata info:
* "mountain pass" : https://www.wikidata.org/wiki/Q133056

* basic WikidataQuery  ( MountenPass AND has a coordinate! )
   * https://w.wiki/tdv  +  "Execute Query" ! 
      * now:20631 results ( needs more cleaning; deduplication , ... )
       * with optional: English wikipedia links  (`?articleEN`) ; mountainRange ; Elevation , Country  )     
      * adapt/modify - and press `</>Code>`  or Download 
![image](https://user-images.githubusercontent.com/2217320/104019201-6fb81000-51bb-11eb-9c15-6059e5992d46.png)
      
* `wdtaxonomy Q133056`  - imho:  useless .. 
```
mountain pass (Q133056) •81 ×21460 ↑↑
├──silniční sedlo (Q12054251) •1
├──Passhöhe (Q14524467) •2 ×2
├──road mountain pass (Q43675388) ×82
│  └──Tōge (Q10383863) •6
├──rail mountain pass (Q46112269) ×48
└──Talwasserscheide (Q85871133) ×1 ↑
```

Map view:
![image](https://user-images.githubusercontent.com/2217320/104020392-6c258880-51bd-11eb-94c5-89b2777e8244.png)

**Comment by Andrettin** [comment]
I found this book with some geocoordinate information on mountains passes:

https://books.google.at/books?id=JHbXAAAAMAAJ&pg=PA342&dq=mountain+pass+latitude&hl=de&sa=X&ved=2ahUKEwiRy-by24vuAhWGy4UKHUjSDFcQ6AEwAXoECAQQAg#v=onepage&q=mountain%20pass%20latitude&f=false

I was thinking of entering its data manually in a geodata file, but I suppose the wikidata query by @ImreSamu could collect that information much more efficiently?

**Comment by ImreSamu** [comment]
@Andrettin 

I don't know what is the best method ..  but in the end we should match the results with the Wikidata  ( so need a WikidataID )  

imho:  the original "20631" record is too much ... for manual checking 
Maybe we can limit, and keeping only the wikidata records -  with English Wikipedia page.
so _" mountainPass AND has an english wikipedia page AND has a Coordinate! "_ 
 *  https://w.wiki/tzZ  this is:  **~1558 record**  ...  probably need more filtering ..

**Comment by nvkelso** [comment]
This should probably number in low hundreds of features, some small fraction of mountain peak count?

**Comment by Maxszik** [comment]
Hello,

_Great idea! Do you have any data or research time to contribute for mountain passes?_

I definitely have the time and desire to contribute. Experience and skills, unfortunately, no. I have no idea about the methodology and how to process data. If someone could point me to some instructions that would be great.

I agree with @nvkelso that the above example with all those wikidata passes is too much. This is just my personal preference, of course. I imagine something like the mountain peaks example, which would be not all passes but those one would consider "epic" or "classic". Totally subjective, obviously.

What about this: there a Wikipedia entry called [List of mountain passes](https://en.wikipedia.org/wiki/List_of_mountain_passes). What if I should begin there, assembling a csv with coordinates and a number of features? Is that a good idea?

**Comment by nvkelso** [comment]
That list looks legit, if too exhaustive (almost 1000 passes). BTW, the red links don't have a full page, the blue links do. That's sometimes a good indicator for how "significant" the feature actually is.

I'd cross reference this list with lists of "top 10 passes" (or 20) by continent and see what's common.

**Comment by ImreSamu** [comment]
The good news:   There are already 2 "mountain pass"  in the NE database.  { `Khyber Pass`,`Chilkoot Pass`} 
*  ` 10m_physical/ne_10m_geography_regions_elevation_points`    ...  `where featurecla='pass' `

|   ne_id    | min_zoom |     name      | featurecla | elevation |    region     | wikidataid | wdid_score |    name_en    |     name_fr     |    name_de    |
|------------|----------|---------------|------------|-----------|---------------|------------|------------|---------------|-----------------|---------------|
| 1159105727 |        5 | Khyber Pass   | pass       |      1070 | Asia          | [Q191333](https://www.wikidata.org/wiki/Q191333)    |          4 | Khyber Pass   | Passe de Khyber | Chaiber-Pass  |
| 1159106213 |      6.7 | Chilkoot Pass | pass       |      1067 | North America | [Q1072958](https://www.wikidata.org/wiki/Q1072958)   |          2 | Chilkoot Pass | col du Chilkoot | Chilkoot Pass |
